packages/flutter_tools/test/general.shard/macos/xcode_test.dart

```diff
@@ -799,6 +828,7 @@ void main() {
 
           final List<String> errors = await xcdevice.getDiagnostics();
           expect(errors, hasLength(4));
+          // Does not contain "Xcode will continue when iPad is connected."
```

Comment:
Can you write an expectation for this? Maybe something like:

`expect(errors, isNot(contains('Xcode will continue when iPad is connected.')));`